src/cascadia/TerminalSettingsEditor/Resources/en-US/Resources.resw

```diff
@@ -1225,7 +1233,7 @@
     <value>Rename</value>
     <comment>Text label for a button that can be used to begin the renaming process.</comment>
   </data>
-  <data name="ColorScheme_DeleteButtonDisclaimerInBox.Text" xml:space="preserve">

```

Comment:
Please keep this key exactly as-is. If we need to change it, duplicate it with the new name.

File a bug for removing it later (incl. any other keys that this PR would have removed.)

Reply:
Do we even need to change this one? It looks like it's been outmoded by the two new separate ones.